fe/fe-core/src/main/java/com/starrocks/planner/OlapTableSink.java

```diff
@@ -345,9 +347,22 @@ private TOlapTableLocationParam createLocation(OlapTable table) throws UserExcep
                                             + tablet.getId() + ", backends: " +
                                             Joiner.on(",").join(localTablet.getBackends()));
                         }
-                        // replicas[0] will be the primary replica
+
                         List<Long> replicas = Lists.newArrayList(bePathsMap.keySet());
-                        Collections.shuffle(replicas);
+                        int lowUsageIndex = 0;
```

Comment:
Is this to pick out the BE with the least primaries? Maybe it's better to add some comments to make it clear.

Reply:
There will be a PR to implement the backlist strategy soon, comment will be made at that time.